Assigment3/EmployeeManagement.java

```diff
@@ -0,0 +1,307 @@
+import java.io.*;
+import java.util.*;
+
+/**
+ * EmployeeManagement class
+ * empList - stores the all data of employee from emp.csv
+ * empFile - BufferedReader instance used to read data from .csv
+ * latestIdAllocated - which initially set to last id allocated to employee
+ * location - represents file location initialized in constructor
+ * saveEmp - used to save data in file .csv
+ * changeStatus - it used to detect is there any changes made in data by add/delete/update operation
+ */
+class EmployeeManagement{
+    private HashMap<Integer,Employee> empList;
+    private BufferedReader empFile;
+    private int latestIdAllocated;
+    private File location;
+    private BufferedWriter saveEmp;
+    private boolean changeStatus; 
+
+    EmployeeManagement(){
+        changeStatus=false;
+        location=new File("./emp.csv");
```

Comment:
please use proper indentation for = like location=new File("./emp.csv"); should be location = new File("./emp.csv");